andrea/js/app.js

```diff
@@ -0,0 +1,96 @@
+// buttons //
+let signUpButton = document.querySelector("#navbar-sign");
+let logInButton = document.querySelector("#navbar-login");
+
+let createButton = document.querySelector(".create-btn")
+
+let welcomeView = document.querySelector(".welcome-view")
+
+// form //
+let form = document.querySelector(".register-form");
+let registerTitle = document.querySelector(".register-title");
+let logInTitle = document.querySelector(".login-title");
+
+let registerFields = Array.from(document.querySelectorAll(".register-fields"));
+
+
+let logInForm = document.querySelector(".login-form");
+let formAlternative = document.querySelector(".form-alternative");
+
+let registerHide = [welcomeView, signUpButton, logInButton];
+let logInHide = [registerTitle, signUpButton, logInButton, createButton, welcomeView, formAlternative];
+logInHide = logInHide.concat(registerFields);
+
+// passwords //
+let password = document.getElementById("password");
+let confirmPassword = document.getElementById("confirm-password");
+
+// change visibility
+function visibilityToggle(element, method, classname){
+    if (Array.isArray(element)){
+        element.forEach(element => {
+            method === "add" ? element.classList.add(classname) : element.classList.remove(classname)
+        })
+    }else{
+        method === "add" ? element.classList.add(classname) : element.classList.remove(classname)
+    }
+
+}
```

Comment:
This function has line 32 and 35 to be the same, violating DRY principle again, can you think of a way to not repeat the same line of code twice inside this function?